crate.ts

```diff
@@ -83,9 +83,25 @@ export class Crate {
 
   async setVersion(version: string) {
     $.logStep(`Setting ${this.name} to ${version}...`);
-    for (const crate of this.repo.crates) {
-      await crate.setDependencyVersion(this.name, version);
+
+    if (this.repo.folderPath !== this.folderPath) {
```

Comment:
Maybe we should add a comment above the if statement saying what this code does.